src/solver.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const parser = require("./sudokuParser");
+const helper = require("./helper");
+
+function getRelatedCells(cell, sudokuMatrix) {
```

Comment:
When reading body of `solve` function my initial though when reached line with `getRelatedCells` was to look into `helper` module. I was wrong :smile: Maybe you should move it there.